### DISCUSSION TITLE
Welcome to [3TH1C4L] Discussions!

### DISCUSSION THREAD

**Discussion by RPxGoon** [RPxGoon/3TH1C4L-MultiTool#1] [Announcements]
### 👋 Welcome to Discussions! 👋
**This Tool is Currently Focused on Bringing Lightweight, Simple OSINT, Pentesting, Discord, and Other General Purpose Tools Into One Cleanly Organized Place. Please Feel Free to at Any Time:**
  * Share Ideas
  * Report Issues
  * Ask Questions or Ask for Help
  * Request Additional Features
  * Engage With Other Community Members
  * Welcome Others and Help Anyone In Need. Remember That This is a Community We
  Build Together 💪

> This Tool Will ALWAYS BE FREE. I Appreciate Any Support/Help Given to the Project or the Community Around It

**Comment by wWezz** [comment]
Can this tool be run in Termux?

**Reply by RPxGoon** [reply]
**> Can this tool be run in Termux?**

_honestly, i would assume so but i haven't tested with termux and definitely didn't implement anything specifically for it, however i will definitely do some testing on my own and ensure it works properly there too, thank you and sorry it took so long to respond. should have this tested and working soon(in ab a day or 2) if its not already!_
